pkg/types/relayer.go

```diff
@@ -99,6 +106,8 @@ type ChainService interface {
 type Relayer interface {
 	ChainService
 
+	NewEVMChain(ctx context.Context) (EVMChain, error)
```

Comment:
Do we need to construct instances of these? Or could the Relayer have the methods embedded? 
```suggestion
	EVMChain
```